MetaMask/metamask-extension
# Fix Sentry console logs

Sentry console logs were failing with the error "Error: TypeError: Cannot read properties of undefined (reading 'log')". This error has been fixed; now Sentry can log errors to the console.

## Manual Testing Steps

* Run `yarn start:test`
* Run any e2e test, see that the error quoted above is not printed to the console.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
